js/views/create_group_dialog_view.js

```diff
@@ -105,19 +105,19 @@
 
       this.isAdmin = groupConvo.get('groupAdmins').includes(ourPK);
 
-      const convos = window.getConversations().models;
-
-      let allMembers = convos.filter(d => !!d);
-      allMembers = allMembers.filter(
-        d => d.isFriend() && d.isPrivate() && !d.isMe()
-      );
-      allMembers = _.uniq(allMembers, true, d => d.id);
+      const convos = window.getConversations().models.filter(d => !!d);
 
-      this.friendList = allMembers;
-
-      // only give members that are not already in the group
       let existingMembers = groupConvo.get('members');
 
+      // Show a contact if they are our friend or if they are a member
+      let friendsAndMembers = convos.filter(
```

Comment:
Does the `const convos` variable is used somewhere else? if not, we could remove this temp list and just do:

```
let friendsAndMembers = window.getConversations().models.filter(
           d =>
          !!d && (d.isFriend() || existingMembers.includes(d.id)) &&
          d.isPrivate() &&
          !d.isMe()
      );
```

Reply:
Not sure if it is an improvement, but I can remove `convos` if you want.

Reply:
Just realized the double filter() was already there before your changes. so we are good to go as is I would say